app/javascript/components/types.ts

```diff
@@ -655,3 +655,8 @@ export type Notification = {
 }
 
 type NotificationImageType = 'icon' | 'avatar'
+
+export type GitFile = {
```

Comment:
There's not really anything specific to Git for this file. Maybe call this `FileWithContents` or `NamedFile` or something else?

Reply:
Ah. Good point. For this, I will be very specific and call it `ExemplarFile` for now. We can then generalize when we have more examples.